packages/p2p/src/components/empty/empty.scss

```diff
@@ -6,19 +6,25 @@
     margin-top: 4rem;
     max-width: 628px;
 
-    &--no-tabs {
-        margin-top: calc(4rem + 4.4rem);
+    &__button {
+        text-decoration: none;
+        margin-top: 1.6rem;
     }
+
     &-icon {
         margin-bottom: 2.4rem;
     }
-    &-title {
-        color: var(--text-general);
-        font-size: var(--text-size-s);
-        font-weight: bold;
-        line-height: 1.5;
-        margin: 0.8rem;
+
+    &--mobile {
+        height: unset;
+        margin-top: 6rem !important;
+        max-width: 100vw;
+    }
+
+    &--no-tabs {
+        margin-top: calc(4rem + 4.4rem);
```

Comment:
:thinking:

Reply:
This is to incorporate the tab text margin as well as the actual `margin-top` from what I gather.